Aardvark/ARKEmailBugReporter.h

```diff
@@ -73,6 +88,9 @@
 /// The email attachment delegate, responsible for providing additional attachments and filtering which log stores to include in the bug report at the time the bug is filed.
 @property (nullable, nonatomic, weak) id <ARKEmailBugReporterEmailAttachmentAdditionsDelegate> emailAttachmentAdditionsDelegate;
 
+/// The prompting delegate, responsible for showing a prompt to file a bug report.
```

Comment:
is there a default behavior if/when this is `nil`?

Reply:
Yep, default behavior is the existing behavior (showing an alert asking for the bug title).  Added to comment.